code-style/action.yml

```diff
@@ -27,6 +27,20 @@ inputs:
     default: true
     type: boolean
 
+  docker-recursive:
+    description: >
+      Search for Dockerfile(s) recursively. Only applies to directory 'docker'.
```

Comment:
This is false, because you can also analyse `.devcontainer` dir.

Reply:
✔️ done